components/FilterTabType/FilterTabType.js

```diff
@@ -30,7 +30,9 @@ export default class FilterTabType extends PureComponent<void, Props, void> {
                 style={styles.accountIssuesType}
                 onPress={onPress}
             >
-                <Text style={styles.accountIssuesTypeText}>{title}</Text>
+                <Text style={styles.accountIssuesTypeText}>
+                    {title.replace(/\s\s/g, '\n')}
```

Comment:
All right, let's leave so? As I mentioned, PR are very similar to issues, because of what a lot of copy-paste, with this something can be done or left so?